### ISSUE TITLE
Disappearance of XML comments in egXML HTML

### ISSUE THREAD

**Issue by TomazErjavec** [TEIC/Stylesheets#564]
Up to, at least, 2022-05-23 the XML comments in egXML were displayed in the HTML, cf. [this page](https://clarin-eric.github.io/parla-clarin/#sec-facsimile) and [corresponding XML](https://github.com/clarin-eric/parla-clarin/blob/7524d09084655644677ae2996d6d2b0f773d28f7/Schema/parla-clarin-odd.xml#L1833-L1840). But using the latest dev release of the TEI Stylesheets they now seem to be gone from the HTML output of egXML - can they be brought back please?

**Comment by lb42** [comment]
I cannot reproduce this, though I have noticed that the formatting of egXML content with the most recent Stylesheet version seems to have changed in other respects

**Comment by sydb** [comment]
@TomazErjavec — I think the short answer is “yes”, if they are missing they can be brought back. (I am not nearly so confident we can get that done by the upcoming release, which is scheduled for Sep or Oct, I think.) However, like @lb42, when I produce local HTML from (a mildly modified version of) that ODD, I see comments in the examples.
What process are you using to generate HTML?

**Comment by TomazErjavec** [comment]
Looking again - after some time - at [the page referred to above](https://clarin-eric.github.io/parla-clarin/#sec-facsimile), which has been re-built  since I posted this issue, I now see that they are now no longer missing but are back in all their glory. I am certain that they were gone when I reported this but also certain they are back now, so this seems to be a non-issue, but rather a temporary glitch somewhere. Thanks for responding anyway, an closing this issue.
